himl/custom_merger_example.py

```diff
@@ -0,0 +1,29 @@
+import argparse
+import os
+from copy import deepcopy
+
+from deepmerge import STRATEGY_END
+
+"""
+Example function to demonstrate how to provide a custom merging behavior to himl.
+How to use it:
+        config_processor = ConfigProcessor()
+        config_processor.process(cwd, opts.path, filters, excluded_keys, opts.enclosing_key, opts.remove_enclosing_key,
+                                 opts.output_format, opts.print_data, opts.output_file, opts.skip_interpolation_resolving,
+                                 opts.skip_interpolation_validation, opts.skip_secrets, opts.multi_line_string,
+                                 type_strategies= [(list, [strategy_merge_override,'append']), (dict, ["merge"])] )
+"""
+def strategy_merge_override(config, path, base, nxt):
```

Comment:
I see this is described very well in the [README.md](https://github.com/adobe/himl/pull/104/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R365). Is there a reason for keeping here as well or we can remoce it?
I'm thinking this file will be bundled in the python package and my concern is that we are polluting it with code that's not actually used. What do you think?

Reply:
kept it just in case people don't read README :) 
I do agree we shouldn't keep it there to keep bundle clean and mean. Removed.

There's a test_config_generator.py in tests folder. As far as I see, it's not used.  Should I remove that as well?